miner/algo_common.go

```diff
@@ -284,13 +295,23 @@ func (envDiff *environmentDiff) commitBundle(bundle *types.SimulatedBundle, chDa
 		receipt, _, err := tmpEnvDiff.commitTx(tx, chData)
 
 		if err != nil {
-			log.Trace("Bundle tx error", "bundle", bundle.OriginalBundle.Hash, "tx", tx.Hash(), "err", err)
-			return err
-		}
+			isRevertibleTx := bundle.OriginalBundle.RevertingHash(txHash)
+			// if drop enabled, and revertible tx has error on commit, we skip the transaction and continue with next one
+			if algoConf.DropRevertibleTxOnErr && isRevertibleTx {
+				log.Info("Found error on commit for revertible tx, but discard on err is enabled so skipping.",
```

Comment:
why is this log.Info? other logs is Trace

Reply:
@Ruteri suggested `log.Info` because this is expected behavior, previous discussion: https://github.com/flashbots/builder/pull/81#discussion_r1252676030

Reply:
For the record, I proposed having it at Info rather than Warn - I'd be fine with debug/trace

Reply:
updated to debug level - this log shouldn't occur very often given current data (~10% requests specify canRevert) and doesn't offer verbose info like trace, so set to debug for now

Reply:
will revisit if things change

Reply:
i deployed for testing and observed that debug log for this actually happens a lot - updated  to trace as @avalonche originally suggested